pkg/util/util.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 The Cluster Monitoring Operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"regexp"
+	"strings"
+)
+
+var (
```

Comment:
This was borrowed from https://github.com/openshift/ocp-release-operator-sdk/blob/419cfd39bea8657babc55759db5fd77cada54334/pkg/ansible/paramconv/paramconv.go but they had no tests there, so decided to add some unit tests and copy it over. It would not make sense I think to import that operator 🤷‍♀ up to you.

Reply:
a little copying is definitely better than a little dependency here :+1: Awesome, that you added tests (maybe contribute those back?)